src/html.ts

```diff
@@ -1,25 +1,30 @@
-import { Doc, Reference, Footnote, Link, HasChildren,
-         HasAttributes, AstNode } from "./ast";
+import {
+  AstNode,
+  Doc,
+  Footnote,
+  HasAttributes,
+  HasChildren,
+  Link,
+  Reference,
+} from "./ast";
 import { getStringContent, ParseOptions } from "./parse";
 
-const defaultWarnings = function(message: string, pos?: number | null) {
+const defaultWarnings = function (message: string, pos?: number | null) {
   console.log(message + (pos ? " at " + pos : "") + "\n");
-}
+};
```

Comment:
Sorry for the formatting changes, accidentally hit reformat in the middle of the exercise, would back those out if we intend to merge this.